### PR TITLE
fix: removed nats box from depoyment

### PR DESCRIPTION
**Pull request by exu**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: charts/testkube/values.yaml [hunk 1]
```diff
@@ -139,11 +139,11 @@ mongodb:
...
-#  tolerations:
-#    - key: kubernetes.io/arch
-#      operator: Equal
-#      value: arm64
-#      effect: NoSchedule
+  #  tolerations:
+  #    - key: kubernetes.io/arch
+  #      operator: Equal
+  #      value: arm64
+  #      effect: NoSchedule
```
Comment (dejanzele) [inline charts/testkube/values.yaml:146]: can you revert this change?

**Comment by exu** [comment]
@dejanzele rolled back

**Review by dejanzele — APPROVED** [review]
